### DISCUSSION TITLE
Is Arcus.WebApi.Logging.AzureFunctions still relevant ?

### DISCUSSION THREAD

**Discussion by fgheysels** [arcus-azure/arcus.webapi#299] [Q&A]
There exists a package 'Arcus.WebApi.Logging.AzureFunctions' and I wonder if this is still relevant ?
Why do we have a package that is related to Functions in the WebApi repository ?  

This package contains only a single extension method `AddHttpCorrelation`  which in turn calls an obsolete method.

![image](https://user-images.githubusercontent.com/3605786/154232538-d7c20eed-ffd9-4437-9b4b-1337d420f430.png)

**Comment by stijnmoreels** [comment]
See #298 for the explanation. Yes, it's still relevant.

**Reply by fgheysels** [reply]
I wonder if that package is in the correct repo then ?  What does it really provide ?  Is it usable for all kind of Azure Functions , or is it only useable for HTTP triggered functions ?

I stumbled upon it when creating this issue https://github.com/arcus-azure/arcus.observability/issues/336

**Reply by stijnmoreels** [reply]
The extension (should) registers the HTTP-specific `HttpCorrelationInfo` model as a correlation accessor service. This model is defined in this repo, so it would have to reference this repo in any case. Also, like we use ASP.NET Core features in a HTTP-trigger, we use here a Web API model in a HTTP-trigger. So, I would think that it lives in the right repo, no?
